task/backend/scheduler/scheduler.go

```diff
@@ -0,0 +1,78 @@
+package scheduler
+
+import (
+	"context"
+	"time"
+
+	"github.com/influxdata/influxdb"
+)
+
+type Storage interface {
+	// UpdateLatestCompleted will allow us to set a latest completed time for an id.
+	UpdateLatestCompleted(ctx context.Context, id influxdb.ID, lastRun time.Time) error
```

Comment:
What do you think of the name `SetCompletionTime`?  That way it would include units.  Also, does that fact that it is the "latest" matter to the users of storage?

Reply:
the "storage" may not need to know its the "latest" completed. so your thinking `CompletionTime` and `SetCompletionTime`?